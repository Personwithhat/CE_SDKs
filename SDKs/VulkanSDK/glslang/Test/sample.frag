version https://git-lfs.github.com/spec/v1
oid sha256:aea96d3f2a22cd3692afed87e0b701bd2c1883758f89585e00ebba0c92c710f0
size 1672
