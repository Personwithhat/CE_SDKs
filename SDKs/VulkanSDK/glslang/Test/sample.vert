version https://git-lfs.github.com/spec/v1
oid sha256:a76bd84b6d0bc9b0fbf8ed260d467c1b98ff1aab7c9ecd5dfb55012ffba21131
size 1729
