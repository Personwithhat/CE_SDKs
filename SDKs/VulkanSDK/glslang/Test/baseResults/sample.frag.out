version https://git-lfs.github.com/spec/v1
oid sha256:a2c2fe6ae62b93f6737527074dfbaa2f5d038289b1f12fb02fc230572e0f7f20
size 1116
